docs/pages/setup/admin/trustedclusters.mdx

```diff
@@ -4,58 +4,59 @@ description: How to configure access and trust between two SSH and Kubernetes en
 h1: Trusted Clusters
 ---
 
-The design of trusted clusters allows Teleport users to connect to compute infrastructure
-located behind firewalls without any open TCP ports. The real-world usage examples of this
-capability include:
+The design of Trusted Clusters allows Teleport users to connect to compute
+infrastructure located behind firewalls without any open TCP ports. The
+real-world usage examples of this capability include:
 
 - Managed service providers (MSP) remotely managing the infrastructure of their clients.
-- Device manufacturers remotely maintaining computing appliances deployed on-premises.
-- Large cloud software vendors manage multiple data centers using a common proxy.
+- Device manufacturers remotely maintaining computing appliances deployed on premises.
+- Large cloud software vendors managing multiple data centers using a common proxy.
 
-**Example of a MSP provider using trusted cluster to obtain access to clients clusters.**
+Here is an example of an MSP using Trusted Clusters to obtain access to client clusters:
 ![MSP Example](../../../img/trusted-clusters/TrustedClusters-MSP.svg)
 
-The Trusted Clusters chapter in the Admin Guide
-offers an example of a simple configuration which:
+This guide will explain how to:
 
-- Uses a static cluster join token defined in a configuration file.
-- Does not cover inter-cluster Role-Based Access Control (RBAC).
-
-This guide's focus is on more in-depth coverage of trusted clusters features and will cover the following topics:
-
-- How to add and remove trusted clusters using CLI commands.
+- Add and remove Trusted Clusters using CLI commands.
 - Enable/disable trust between clusters.
 - Establish permissions mapping between clusters using Teleport roles.
 
-<Admonition
-  type="tip"
-  title="Teleport Node Tunneling"
->
-  If you have a large number of devices on different networks, such as managed IoT devices or a couple of nodes on a different network you can utilize [Teleport Node Tunneling](./adding-nodes.mdx).
-</Admonition>
+<Details title="Teleport Node Tunneling" scope={["enterprise", "oss"]} scopeOnly opened>
 
-## Introduction
+ If you have a large number of devices on different networks, such as managed
```

Comment:
I'm not sure the criteria for using tunneling is "a large number of devices."

Technically, node tunneling consumes **more** resources than direct dial, so your cluster would better handle a large number of devices if you did not use tunneling.

I would recommend we reword this so that networking is the main driver for tunneling. 
Something like:

If your nodes are deployed behind a firewall or otherwise not reachable by the Teleport Proxy Service, you can connect your nodes via Teleport Node Tunneling. Instead of connection to the Auth Service directly, ...